### PR TITLE
Add cmmands, structure them and add headers

### PR DESCRIPTION
**Pull request by KittyBorgX**

### CODE REVIEW

File: src/commands/general/cmd.help.ts [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
+// Maintainer: Shadofer#0001
+// Contributors: Otterlord#3653, KittyBorgX#1415
+//
+// File description:
+//     The file for the !help command of the TowerBot
+// 
+// Extra info:
+//     Displays all the 
+//     Makes request to wss://towerverse-beta.herokuapp.com'
+//
```
Comment (theotterlord) [inline src/commands/general/cmd.help.ts]: 0. Do we really need these in the files?
1. The maintainer is not shadofer
2. Needs to use github usernames

File: src/commands/towerverse/cmd.travellers.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
+// License: GPL-3
+// Maintainer: Shadofer#0001
+// Contributors: Otterlord#3653, KittyBorgX#1415
+//
+// File description:
+//     The file for the !online command of the TowerBot
+//
+// Extra info:
+//     This file is using the towerverse.js npm package.
+//     Makes request to wss://towerverse-beta.herokuapp.com'
```
Comment (theotterlord) [inline src/commands/towerverse/cmd.travellers.ts]: don't embed the url

File: src/commands/towerverse/cmd.travellers.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  name: "travellers",
+  desc: "Shows total number of travellers",
+  args: [{ name: "server", required: false }],
+});
+
+const client = new Client();
+
+command.use(cooldown(10000));
+command.setExecutor(async (app, msg, args) => {
+  client.connect("wss://towerverse-beta.herokuapp.com").then(async () => {
```
Comment (theotterlord) [inline src/commands/towerverse/cmd.travellers.ts]: we should probably be querying the main app

File: src/commands/towerverse/cmd.status.ts [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  });
+}
+
+command.use(cooldown(10000));
+command.setExecutor(async (app, msg, args) => {
+  const pings: any[] = [];
+
+  if (args[0] !== "beta") pings.push(["stable", await ping()]);
+  if (args[0] !== "stable")
+    pings.push(["beta", await ping("wss://towerverse-beta.herokuapp.com")]);
```
Comment (theotterlord) [inline src/commands/towerverse/cmd.status.ts]: should probably ping the default server

File: src/commands/towerverse/cmd.online.ts [hunk 5]
```diff
@@ -0,0 +1,41 @@
+// License: GPL-3
+// Maintainer: Otterlord#3653, KittyBorgX#1415
+// Contributors:
+//
+// File description:
+//     The file for the !status command of the TowerBot
+//
+// Extra info:
+//     This file is using the towerverse.js npm package.
```
Comment (theotterlord) [inline src/commands/towerverse/cmd.online.ts]: I feel this is a little too specific and not needed. just adds confusion to the project

File: src/commands/towerverse/cmd.online.ts [hunk 6]
```diff
@@ -0,0 +1,41 @@
+// License: GPL-3
+// Maintainer: Otterlord#3653, KittyBorgX#1415
+// Contributors:
+//
+// File description:
+//     The file for the !status command of the TowerBot
```
Comment (theotterlord) [inline src/commands/towerverse/cmd.online.ts]: this should be obvious

File: src/commands/towerverse/cmd.travellers.ts [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+    console.log("Connected to server");
+    client
+      .listTravellers()
```
Comment (theotterlord) [inline src/commands/towerverse/cmd.travellers.ts]:
```suggestion
      .totalTravellers()
```

File: src/commands/towerverse/cmd.travellers.ts [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+      .then((data) => {
+        const Embed = new MessageEmbed()
+          .setTitle("Number of online travellers!")
+          .setAuthor(`${msg.author.username}`, msg.author.displayAvatarURL())
+          .setDescription(`Found ${data.length} traveller`)
```
Comment (theotterlord) [inline src/commands/towerverse/cmd.travellers.ts]:
```suggestion
          .setDescription(`Found ${data} traveller`)
```

**Review by theotterlord — CHANGES_REQUESTED** [review]
Overall, it looks good. but there are a few things that need to be dealt with before I can merge

File: src/commands/towerverse/cmd.online.ts [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+import { Message, MessageEmbed } from "discord.js";
+import { Client } from "towerverse.js";
+import { cooldown } from "../../middleware/cooldown";
+import { Command } from "../../structures/Command";
+import { performance } from "perf_hooks";
+
+const command = new Command({
+  name: "online",
+  desc: "Shows number of online travellers",
+  args: [{ name: "server", required: false }],
```
Comment (theotterlord) [inline src/commands/towerverse/cmd.online.ts]: this argument is never used

File: src/commands/towerverse/cmd.online.ts [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+import { MessageEmbed } from "discord.js";
+import { Client } from "towerverse.js";
+import { cooldown } from "../../middleware/cooldown";
+import { Command } from "../../structures/Command";
+
+const command = new Command({
+  name: "online",
+  desc: "Shows number of online travellers",
+  args: [{ name: "server", required: false }],
+});
```
Comment (theotterlord) [inline src/commands/towerverse/cmd.online.ts]: this argument is not used
```suggestion
  desc: "Shows number of online travellers",
});
```

**Review by theotterlord — CHANGES_REQUESTED** [review]

File: src/commands/towerverse/cmd.online.ts (hunk 10 above)
Comment (KittyBorgX) [reply src/commands/towerverse/cmd.online.ts]: done

**Review by theotterlord — APPROVED** [review]

**Pull request merged**